lib/runtime-core/src/memory/static_/unshared.rs

```diff
@@ -0,0 +1,99 @@
+use crate::{
+    memory::{
+        static_::{SAFE_STATIC_GUARD_SIZE, SAFE_STATIC_HEAP_SIZE},
+        WASM_MAX_PAGES, WASM_PAGE_SIZE,
+    },
+    sys,
+    types::MemoryDesc,
+    vm,
+};
+
+pub struct StaticMemory {
+    memory: sys::Memory,
+    current: u32,
```

Comment:
Maybe name this and `current()`  `current_size` `actual_size`, or just `size`

Reply:
Yeah, I wasn't sure what to name the various size functions.

Reply:
The current is the number of pages, not the actual size.

Reply:
Maybe `current_pages` or add some docs

Reply:
The js api just exposes the actual length of the memory, not the number of pages. Should we follow suite?

Reply:
Not exactly sure on this, this API appears to have pages as well: https://github.com/rossberg/wasm-c-api/blob/master/include/wasm.h#L450